package.json

```diff
@@ -191,6 +196,9 @@
   "license": "MIT",
   "main": "lib/index.js",
   "name": "@metamask/ethjs-contract",
+  "peerDependencies": {
+    "@babel/runtime": "^7.0.0"
```

Comment:
Why do we need this?

Reply:
Sorry, where are you looking? I see mention of plugins having a peer dependency on `@babel/core` here: https://babeljs.io/docs/v7-migration#versioningdependencies. But I don't see anything around a need to add Babel as a peer dependency, much less the fact that previous versions implied a peer dependency.

More importantly, we're just using this package as a dev dependency. Why should consumers of this package need to have `@babel/runtime` installed just to use it?